src/ape/managers/chain.py

```diff
@@ -441,6 +441,12 @@ def get(
         contract_type = self._get_contract_type_from_disk(address)
 
         if not contract_type:
+            # Contract could be a minimal proxy
+            if hasattr(self.provider.network.ecosystem, "proxy_info"):
+                proxy = self.provider.network.ecosystem.proxy_info(address)
```

Comment:
Should definitely add this method to `EcosystemAPI` abstract base class with a default implementation that returns empty list

Reply:
Yes, then you don't have to use `hasattr()`, the method will just always exists.

Reply:
Exactly!